cur.php

```diff
@@ -0,0 +1,271 @@
+        <a class="navbar-brand navbar-logo" href="index.php">Measurement Converter</a>
+        <button class="navbar-toggler" type="button" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <i class="fas fa-bars text-white"></i>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ml-auto">
+                <div class="hori-selector">
+                    <div class="left"></div>
+                    <div class="right"></div>
+                </div>
+                <li class="nav-item ">
+                    <a class="nav-link" href="index.php"><i class="fa fa-address-card"></i>About</a>
+                </li>
+                <li class="nav-item ">
+                    <a class="nav-link" href="fb.php"><i class="fa-regular fa-envelope"></i>Feedback</a>
+                </li>
+                <li class="nav-item ">
+                    <a class="nav-link" href="ld.php"><i class="fa fa-ruler"></i>Length & Distance</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="area.php"><i class="fa fa-clone"></i>Area</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="vol.php"><i class="fa fa-prescription-bottle"></i>Volume and Capacity</a>
+                </li>
+                <li class="nav-item ">
+                    <a class="nav-link" href="mass.php"><i class="fa-solid fa-scale-unbalanced-flip"></i>Mass and Weight</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="speed.php"><i class="fa fa-tachometer-average"></i>Speed</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="temp.php"><i class="fa fa-temperature-high"></i>Temprature</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="cur.php"><i class="fa fa-thin fa-wallet"></i>Currency</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <hr>
+    <div class="container">
+
+        <div class="main">
+
+            <div class="form-group">
+                <label for="oamount">
+                    Amount to Convert :
+                </label>
+                <input type="text" class="form-control searchBox" placeholder="0.00" id="oamount">
+            </div>
+
+            <div class="row">
+                <div class="col-sm-6">
+                    <div class="input-group mb-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text">From</span>
+                        </div>
+                        <select class="form-control from" id="sel1">
+                            <option value="">Select One …</option>
+                            <option value="USD">USD</option>
```

Comment:
Instead of adding static values in dropdown, you can create an array in php file and iterate here using foreach loop.
For eg. create a php file like currency.php and create an array there. in include that file here and use it here for iteration here.